Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -277,7 +278,16 @@ static void softlinkDataDetectorsFrameworks()
 
 #if HAVE(VIDEO_RESTRICTED_DECODING)
 #if PLATFORM(MAC)
-    SandboxExtension::consumePermanently(parameters.trustdExtensionHandle);
+    if (SandboxExtension::consumePermanently(parameters.trustdExtensionHandle)) {
+        // Open up a Mach connection to trustd by doing a code check validation.
+        // This is required since launchd will be blocked after process launch, which prevents new Mach connections to be created.
+        auto bundleURL = adoptCF(CFURLCreateWithFileSystemPath(kCFAllocatorDefault, CFSTR("/System/Library/Video/Plug-Ins/AppleGVAHEVCDecoder.bundle"), kCFURLPOSIXPathStyle, false));
```

Comment:
This seems very fragile and not future-proof. What if that path changes, or some other plugin starts to be used for some image types?

Reply:
That is a good point, Simon. In this case, the actual path used isn't important, since the purpose is to just open up a connection to trustd. I will add a comment and a FIXME about this.

Thanks for reviewing!

Reply:
If AppleGVAHEVCDecoder.bundle is not present on the system will trustd still launch to verify its signature?

Reply:
> If AppleGVAHEVCDecoder.bundle is not present on the system will trustd still launch to verify its signature?

Good point! I will have to check that manually by specifying a non-existent path.

Thanks for reviewing!

Reply:
> > If AppleGVAHEVCDecoder.bundle is not present on the system will trustd still launch to verify its signature?
> 
> Good point! I will have to check that manually by specifying a non-existent path.
> 
> Thanks for reviewing!

Good catch! I have changed the bundle URL to address this.